modules/installation-configuration-parameters.adoc

```diff
@@ -1727,6 +1735,46 @@ Additional Azure configuration parameters are described in the following table:
 |Defines the type of disk.
 |`premium_LRS` or `standardSSD_LRS`. The default is `premium_LRS`.
 
+|`controlPlane.platform.azure.type`
+|Defines the azure instance type for control plane machines.
+|String
+
+|`controlPlane.platform.azure.zones`
+|The availability zones where the installation program creates control plane machines.
+|String list
+
+|`platform.azure.defaultMachinePlatform.encryptionAtHost`
+|Enables host-level encryption for compute machines. You can enable this encryption alongside user-managed server-side encryption. This feature encrypts temporary, ephemeral, cached and un-managed disks on the VM host. This is not a prerequisite for user-managed server-side encryption.
+|`true` or `false`. The default is `false`.
+
+|`platform.azure.defaultMachinePlatform.osDisk.diskEncryptionSet.name`
+|The name of the disk encryption set that contains the encryption key from the installation prerequisites.
+|String, for example `production_disk_encryption_set`.
+
+|`platform.azure.defaultMachinePlatform.osDisk.diskEncryptionSet.resourceGroup`
+|The name of the Azure resource group that contains the disk encryption set from the installation prerequisites. This resource group should be different from the resource group where you install the cluster to avoid deleting your Azure encryption key when the cluster is destroyed. This value is only necessary if you intend to install the cluster with user-managed disk encryption.
```

Comment:
Apart from the suggestion below, I would still suggest to consider breaking this sentence. This is a complex one.

```suggestion
|The name of the Azure resource group that contains the disk encryption set from the installation prerequisites.  To avoid deleting your Azure encryption key when the cluster is destroyed, this resource group must be different from the resource group where you install the cluster. This value is only necessary if you intend to install the cluster with user-managed disk encryption.
```